test/src/auth/SignInFormPasswordAndCustomField.tsx

```diff
@@ -0,0 +1,39 @@
+import { SignInWithPassword } from "@/auth/SignInWithPassword";
+import { Toaster } from "@/components/ui/toaster";
+import { useState } from "react";
+import { HexColorPicker } from "react-colorful";
+
+export function SignInFormPasswordAndCustomField() {
+  const [color, setColor] = useState<string | null>(null);
+  return (
+    <div className="max-w-[384px] mx-auto flex flex-col gap-4">
+      <h2 className="font-semibold text-2xl tracking-tight">
+        Sign in or create an account
+      </h2>
+      <SignInWithPassword
```

Comment:
I suspect we could refactor this (don't love that the `customSignUp` depends on being rendered within a form), but this is fine for now.

One eventual goal is for the UI for the most common flows to be copy-paste-able (Clerk does a really good job of this with packaged `<SignIn/>` components), so we'll probably do a pass at all the components eventually.

Reply:
Thanks for the context.

Since we want to offer customization of what's included in the sign up flow, having the `customSignUp` be part of the form makes sense to me. Do you have a different structure in mind though? Totally fine reworking it in a follow-up PR.